### ISSUE TITLE
Unsupported React DevTools backend version

### ISSUE THREAD

**Issue by jaril** [RecordReplay/devtools#6249]
```
BenWa — Yesterday at 7:10 AM
Got this report from a user. I haven't tried today. Seems like this error is only relevant for standalone React Devtools. (FWIW we discourage global module installation because it causes hard to reproduce issues with module resolution)
Image
jlast — Yesterday at 7:11 AM
hmm - this issue is often misleading
Can the user consistently reproduce it?
BenWa — Yesterday at 3:39 PM
I didn't hear back from them. I got it today while React DevTools was acting funny but I was able to click past it. Ill record it if I can but it was very random.
```

<img width="718" alt="image" src="https://user-images.githubusercontent.com/15959269/162253534-2b1de518-b2c7-45b8-9eb8-10d33027320e.png">

**Comment by bvaughn** [comment]
@jaril This looks like what happens if React DevTools sends the "_getBridgeProtocol_" message to the backend and [doesn't get a response within 10 seconds](https://github.com/facebook/react/blob/d68b09defca37c30d3a7a34d4de884e374c74858/packages/react-devtools-shared/src/devtools/store.js#L1413-L1421). Could that be possible if the Replay session was super bogged down or something?

Since we (Replay) _know_ the protocol version ahead of time– (in fact, since #6075, we use it to choose which DevTools frontend to load)– we could theoretically do a couple of things:
1. Stash it up front and add a special case to the `async send` override method that just sync returns it rather than going through `ThreadFront.evaluate` again. (Whether this helped would depend on my question above RE: overload).
2. Land a change in React DevTools that adds support to the Store for _disabling_ the protocol version check entirely. (Maybe we should _also_ do this, assuming the React team has no concerns here at least.)

**Comment by bvaughn** [comment]
Related PR for DevTools: https://github.com/facebook/react/pull/24303

**Comment by bvaughn** [comment]
The `Store` change (https://github.com/facebook/react/pull/24303) has been published to NPM as version 4.24.4 (https://github.com/facebook/react/pull/24315). I'll start the upgrade process now.

**Comment by bvaughn** [comment]
#6275 will fix this.
